packages/redis/src/Utils.js

```diff
@@ -48,3 +48,11 @@ export const assertSetArgs = (key, value) => {
     `Make sure both key and value are strings for set(): e.g., set('name', 'fela')`
   );
 };
+
+export const assertjGetArgs = key => {
+  assertArgType(
+    key,
+    'string',
+    `Make sure to pass a strings for jGet(): e.g., jGet('patient').`
```

Comment:
`pass a string` (string singular) would be better english. And don't end error/status/console messages in full stops please.

Reply:
Fixed the typo